### ISSUE TITLE
Training on Crowdhuman

### ISSUE THREAD

**Issue by anuragdeshmukh** [WongKinYiu/ScaledYOLOv4#79]
I tried training Yolov4-P7 on crowdhuman with a batch size of 16 and learning rate 0.0025 (2GPUs).
The mAP on val set is flattening at 0.697 after epoch 11 and 12  and then it started lowering to **0.63** after epoch 13.

Am I keeping some hyperparameter wrong?

Has anyone managed to train on crowhuman?

**Comment by WongKinYiu** [comment]
it normal.
![image](https://user-images.githubusercontent.com/12152972/101458056-4cf2cb80-3971-11eb-8b33-8df9fb18242c.png)

by the way, for high density object detection, i suggest to modify the [match function](https://github.com/WongKinYiu/ScaledYOLOv4/blob/yolov4-large/utils/general.py#L529-L533) for build targets.
for example, max wh_iou, wh_iou > 0.7 ...

**Comment by bobbilichandu** [comment]
I am facing same issue, while training on crowdhuman. @WongKinYiu Can you share a sample code snippet which can explain how to modify the match function?

**Comment by WongKinYiu** [comment]
example is already provided in [general.py](https://github.com/WongKinYiu/ScaledYOLOv4/blob/yolov4-large/utils/general.py#L532)
you may also need to modify the [metric function](https://github.com/WongKinYiu/ScaledYOLOv4/blob/yolov4-large/utils/general.py#L933) to do same thing.
